typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -399,6 +403,11 @@ export interface MapProps {
      * The reference to the deck.gl instance.
      */
     deckGlRef?: React.ForwardedRef<DeckGLRef>;
+
+    /**
+     * Callback called from deck.gl whith metrics data.
+     */
+    onMetrics?: ((m: DeckMetrics) => void) | null;
```

Comment:
Suggest using something like `DeckGL["_onMetrics"]`, so that we don't need to duplicate the type